src/lib/modules/Document/DocumentCard/DocumentCard.js

```diff
@@ -75,6 +75,7 @@ class DocumentCard extends Component {
                 {metadata.publication_year}
                 {metadata.edition && <> - Edition {metadata.edition}</>}
                 {volume && <> - Vol. {volume}</>}
+                {metadata.imprint?.publisher && <> <br></br> {metadata.imprint?.publisher}</>}
```

Comment:
```suggestion
                {metadata.imprint?.publisher && <> <br></br> {metadata.imprint.publisher}</>}
```
Not needed second check

Reply:
Right, thanks.